attiny.go

```diff
@@ -112,6 +119,27 @@ func (a *attiny) PingWatchdog() error {
 	return a.write(watchdogReg, nil)
 }
 
+func (a *attiny) UpdateWifiState() error {
+	outByte, err := exec.Command("ip", "a", "show", wifiInterface).Output()
+	if err != nil {
+		return err
+	}
+	newState := strings.Contains(string(outByte), "state UP")
+	if a.wifiConnectedState == newState {
```

Comment:
I'm pretty sure you need to protect access to `a.wifiConnectedState` with a mutex as `UpdateWifiState` can get called concurrently via the D-Bus API. Perhaps just have a mutex for the whole call?